foreign_cc/configure.bzl

```diff
@@ -60,12 +63,31 @@ def _create_configure_script(configureParameters):
     attrs = configureParameters.attrs
     inputs = configureParameters.inputs
 
-    install_prefix = _get_install_prefix(ctx)
+    if ctx.attr.install_prefix:
+        if ctx.attr.install_prefix[0] == "/":
+            install_prefix = ctx.attr.install_prefix
```

Comment:
Setting prefix seems to cause issues with the output directories and the paths it searches, need to investigate this properly